### PR TITLE
Feature-Aaron:assessment-email-template

### PR DESCRIPTION
**Pull request by twikista**

### CODE REVIEW

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  xmlns:o="urn:schemas-microsoft-com:office:office"
+>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="x-apple-disable-message-reformatting" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&family=Pacifico&family=Roboto+Condensed:wght@400;700&display=swap"
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:14]: Font family is `Lexend`

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+<html
+  lang="en "
+  xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:o="urn:schemas-microsoft-com:office:office"
+>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="x-apple-disable-message-reformatting" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:11]: `<link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link
      href="https://fonts.googleapis.com/css2?family=Lexend:wght@100;200;300;400;500;600;700;800;900&display=swap"
      rel="stylesheet"
    />`

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+      rel="stylesheet"
+    />
+    <title>Assessment Invite</title>
+    <style>
+      table,
+      td,
+      div,
+      h1,
+      p {
+        font-family: 'Roboto Condensed', sans-serif;
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:24]: Please change this to `Lexend`

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+        <td
+          align="center"
+          style="padding: 30px 10px; background-color: #6246ea"
+        >
+          <div
+            style="
+              color: #fffffe;
+              font-size: 42px;
+              font-weight: 700;
+              font-family: 'Comfortaa', cursive;
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:68]: Please change this to `Lexend`

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+        width: 100%;
+        border-collapse: collapse;
+        border: 1px solid #d1d1e9;
+        border-spacing: 0;
+      "
+    >
+      <tr>
+        <td
+          align="center"
+          style="padding: 30px 10px; background-color: #6246ea"
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:61]: change background-color value to `#0066C5`

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+              font-family: 'Comfortaa', cursive;
+            "
+          >
+            talentyard
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td align="center" style="padding: 10px 10px">
+          <h1 style="font-size: 24px; color: #6246ea; font-weight: 700">
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:77]: Please use this as the style for line 77
`style="
              font-size: 24px;
              color: #0066C5;
              font-weight: 400;
              font-family: 'Lexend';
            "`

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+            moves you closer to getting your dream job. Endeavor to take the
+            assessment latest {{date}}.
+          </p>
+          <p style="padding: 10px 0">Good luck!</p>
+        </td>
+      </tr>
+      <tr>
+        <td align="center" style="padding: 0 20px; color: #535356">
+          <button
+            style="
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:98]: Use this for the style of the button
`style="
              font-size: 18px;
              background-color: #0066c5;
              color: #fffffe;
              font-weight: 400;
              font-family: 'Lexend', sans-serif;
              width: 200px;
              border: none;
              padding: 15px 10px;
              margin: 10px 0;
            "`

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+            Go to Assessment
+          </button>
+        </td>
+      </tr>
+      <tr>
+        <td style="padding: 0 20px; color: #535356">
+          <p style="padding: 0"></p>
+        </td>
+      </tr>
+      <tr>
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:118]: Please remove this entire element and the content <tr></tr>

File: app/api/Template/assessmentEmailTemplate.handlebars [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+              padding: 0 0 10px 0;
+              font-family: 'Pacifico', cursive;
+              font-size: 20px;
+            "
+          >
+            team talentyard
+          </p>
+        </td>
+      </tr>
+      <tr>
```
Comment (annonymousauthority) [inline app/api/Template/assessmentEmailTemplate.handlebars:131]: Please change this entire section to this
`<tr>
        <td
          align="center"
          style="padding: 10px 20px; background-color: #0066c5"
        >
          <span
            style="
              color: #fffffe;
              font-size: 14px;
              font-weight: 700;
              font-family: 'Lexend', sans-serif;
            "
          >
            talentyard |
          </span>
          <span style="color: #fffffe; font-size: 14px; font-weight: 700">
            United States
          </span>
        </td>
      </tr>`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
This was perfectly executed,
Just make those minor changes, and tag me afterwards for a review.

File: app/api/Template/assessmentEmailTemplate.handlebars (hunk 2 above)
Comment (twikista) [reply app/api/Template/assessmentEmailTemplate.handlebars:11]: Font family changed to `Lexend`

`<link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link
      href="https://fonts.googleapis.com/css2?family=Lexend:wght@100;200;300;400;500;600;700;800;900&display=swap"
      rel="stylesheet"
    />`
@annonymousauthority

File: app/api/Template/assessmentEmailTemplate.handlebars (hunk 4 above)
Comment (twikista) [reply app/api/Template/assessmentEmailTemplate.handlebars]: Font Family changed to `Lexend`

`
              color: #fffffe;
              font-size: 42px;
              font-weight: 700;
              font-family: 'Lexend', sans-serif;
           `

File: app/api/Template/assessmentEmailTemplate.handlebars (hunk 5 above)
Comment (twikista) [reply app/api/Template/assessmentEmailTemplate.handlebars]: background-color changed to `#0066C5`

`style="padding: 30px 10px; background-color: #0066c5"`

File: app/api/Template/assessmentEmailTemplate.handlebars (hunk 6 above)
Comment (twikista) [reply app/api/Template/assessmentEmailTemplate.handlebars]: suggested styles applied

`<h1
            style="
              font-size: 24px;
              color: #0066c5;
              font-weight: 400;
              font-family: 'Lexend';
            "
          >
            Assessment Invitation
          </h1>`

File: app/api/Template/assessmentEmailTemplate.handlebars (hunk 7 above)
Comment (twikista) [reply app/api/Template/assessmentEmailTemplate.handlebars:105]: suggested button styles applied
@annonymousauthority

File: app/api/Template/assessmentEmailTemplate.handlebars (hunk 8 above)
Comment (twikista) [reply app/api/Template/assessmentEmailTemplate.handlebars]: section and content removed

File: app/api/Template/assessmentEmailTemplate.handlebars (hunk 9 above)
Comment (twikista) [reply app/api/Template/assessmentEmailTemplate.handlebars]: section updated to reflect suggested code

` <tr>
        <td
          align="center"
          style="padding: 10px 20px; background-color: #0066c5"
        >
          <span
            style="
              color: #fffffe;
              font-size: 14px;
              font-weight: 700;
              font-family: 'Lexend', sans-serif;
            "
          >
            talentyard |
          </span>
          <span style="color: #fffffe; font-size: 14px; font-weight: 700">
            United States
          </span>
        </td>
      </tr>`
@annonymousauthority

**Comment by twikista** [comment]
suggestions implemented
@annonymousauthority

**Review by annonymousauthority — APPROVED** [review]
LG

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**